src/methods.jl

```diff
@@ -179,3 +179,72 @@ function trimview(frame::AbstractArray, idxs)
 
     return selectdim(frame, d, complement_idxs)
 end
+
+
+"""
+    crop(frame::AbstractArray, shape; force_equal = true)
+
+Crops the image to the shape specified asumming center as the anchor.
+
+Shape speficies the size of the cropped array, force_equal when set to true forces the cropped rows and columns to be equal on both sides
+otherwise the cropped frame contains one extra column on left and one extra row on top if necessary.
```

Comment:
Thoughts about the documentation?

Reply:
```suggestion
This will remove rows/cols of the `frame` equally on each side. When there is an uneven difference in sizes (e.g. size 9 -> 6 can't be removed equally) the default is to increase the output size (e.g. 6 -> 7) so there is equal removal on each side. To disable this, set `force_equal=false`, which will remove the extra slice from the end of the axis.
```